src/components/Button.jsx

```diff
@@ -1,13 +1,22 @@
-const Button = (props) => {
-  const { id, title } = props;
-  return (
-    <>
-      <button className="bg-blue-400 hover:bg-blue-500 active:bg-red-100 transition p-3 rounded-3xl w-full">
-        <span className="mr-1">{id}.</span>
-        {title}
-      </button>
-    </>
-  );
+import PropTypes from 'prop-types';
+
+const Button = ({ id, handleClick, title, className, disabled = false }) => (
+  <button
+    className={`bg-blue-400 hover:bg-blue-500 active:bg-red-100 transition p-3 rounded-3xl w-full disabled:bg-gray-500 disabled:cursor-not-allowed ${className}`}
+    onClick={handleClick}
+    disabled={disabled}
+  >
+    <span className="mr-1">{id}</span>
```

Comment:
need to include verification if id is presented using ternary operator 
I would suggest to use either title || children